sources/world/Makefile

```diff
@@ -47,21 +47,21 @@ clean: ## Remove output files
 ../../data/admin_regions_lvl2_v2.geo.json: ../../data/admin_regions_lvl2_v2.topo.json ## Regions GeoJSON full size
 	mapshaper -i $(DATA_DIR)/admin_regions_lvl2_v2.topo.json \
   -sort this.properties.region_iso_code \
-  -o rfc7946 id-field=region_iso_code format=geojson force /tmp/admin_regions_lvl2_v2.geo.json
```

Comment:
In [mapshaper 0.5.6](https://github.com/mbloch/mapshaper/releases/tag/v0.5.6) there was this change:

> The old rfc7946 flag is no longer useful, but is retained for backwards compatibility. This flag (as before) truncates coordinates to 6 decimal places. Now, using this flag is equivalent to adding precision=0.000001 to the output command.

Reply:
Thanks for fixing this. I don't agree with Mapshaper that `precision=0.000001` is better than `rfc7946` because it is more verbose and more error prone (easy to introduce typos). But that's just my opinion and we should use their preferred flag.

Reply:
I agree, it makes more sense to me to be explicit about the standard. The way they deprecated it was that the flag did not raise an error, but also had no effect at all! :shrug: